src/iterator.rs

```diff
@@ -138,9 +138,10 @@ impl Drop for RegisteredSignals {
 ///
 /// # `mio` support
 ///
-/// If the crate is compiled with the `mio-support` flag, the `Signals` becomes pluggable into
-/// `mio` (it implements the `Source` trait). If it becomes readable, there may be new signals to
-/// pick up. The structure is expected to be registered with level triggered mode.
+/// If the crate is compiled with the `mio-support` or `mio-0_7-support` flags, the `Signals`
+/// becomes pluggable into `mio` (it implements the `Source` trait). If it becomes readable, there
```

Comment:
Maybe extend it to something like „pluggable into `mio` version `0.6` or `0.7` respectively“